Library/Formula/apt-cacher-ng.rb

```diff
@@ -42,7 +30,9 @@ def install
       s.gsub!(/^LogDir: .*/, "LogDir: #{var}/log")
     end
 
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    args << "-DUSE_SSL=OFF"
```

Comment:
Maybe worth adding a notation that SSL support is supplied by `libtomcrypt` rather than binned entirely.